velero-plugins/buildconfig/restore.go

```diff
@@ -52,7 +53,7 @@ func (p *RestorePlugin) updateSecretsAndDockerRefs(buildconfig buildv1API.BuildC
 		return buildconfig, err
 	}
 
-	secretList, err := client.Secrets(buildconfig.Namespace).List(metav1.ListOptions{})
+	secretList, err := client.Secrets(buildconfig.Namespace).List(context.TODO(), metav1.ListOptions{})
```

Comment:
NIT: if we dont have any intention to use the context here in the future, might be good to have `context.Background()` functionally its the same things as `context.TODO()` but semantically it means that we do not need to use this context in a meaningful way.